### PR TITLE
Invite new members CTA update

### PR DESCRIPTION
**Pull request by jmanhart**
+ Adding a new "Invite New Members" CTA button 
- Removing the alert CTA banner

**Old Layout:** 
Screen Shot 2021-05-25 at 9 14 00 PM

**New Layout:**
Screen Shot 2021-05-25 at 9 13 31 PM

### CODE REVIEW

**Review by scefali — COMMENTED** [review]
@jmanhart How is the old banner being removed? All I see is the button being added.

File: static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx [hunk 1]
```diff
@@ -104,9 +105,20 @@ class OrganizationMembersWrapper extends AsyncView<Props, State> {
...
+        icon={<IconMail />}
+      >
+        {t('Invite New Members')}
+      </Button>
+    );
+
     return (
       <Fragment>
-        <SettingsPageHeader title="Members" />
+        <SettingsPageHeader title="Members" action={action} />
```
Comment (EvanPurkhiser) [inline static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx:121]: Did we delete the banner?

Comment (EvanPurkhiser) [reply static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx:121]: Looks like the alert link is floating down here still

**Review by evanpurkhiser — DISMISSED** [review]
delete alert link

**Comment by jmanhart** [comment]
@EvanPurkhiser & @scefali Banner is removed and the tests are updated for a `StyledButton` should be ready to roll

File: static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx [hunk 2]
```diff
@@ -104,18 +104,20 @@ class OrganizationMembersWrapper extends AsyncView<Props, State> {
...
     const {requestList, inviteRequests} = this.state;
 
+    const action = (
+      <Button
+        priority="primary"
+        size="small"
+        onClick={() => openInviteMembersModal({source: 'members_settings'})}
+        icon={<IconMail />}
+      >
+        {t('Invite New Members')}
```
Comment (ckj) [inline static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx:114]: Thoughts on just "Invite member" or "Invite members" as the cta?

Comment (jmanhart) [reply static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx:114]: err I mean kill the new

**Review by scefali — APPROVED** [review]

**Pull request merged**